docs/atlascli/command/atlas-clusters-update.txt

```diff
@@ -100,17 +106,23 @@ Examples
 
 .. code-block::
 
-   # Update tier for a cluster:
-   atlas cluster update <clusterName> --projectId <projectId> --tier M50
+   # Update the tier for a cluster named myCluster for the project with ID 5e2211c17a3e5a48f5497de3:
+   atlas cluster update myCluster --projectId 5e2211c17a3e5a48f5497de3 --tier M50
 
    
 .. code-block::
 
-   # Update disk size for a cluster:
-   atlas cluster update <clusterName> --projectId <projectId> --diskSizeGB 20
+   # Update the disk size for a cluster named myCluster for the project with ID 5e2211c17a3e5a48f5497de3:
+   atlas cluster update myCluster --projectId 5e2211c17a3e5a48f5497de3 --diskSizeGB 20
 
    
 .. code-block::
 
-   # Update MongoDB version for a cluster:
-   atlas cluster update <clusterName> --projectId <projectId> --mdbVersion 4.2
+   # Update the MongoDB version for a cluster named myCluster for the project with ID 5e2211c17a3e5a48f5497de3:
+   atlas cluster update myCluster --projectId 5e2211c17a3e5a48f5497de3 --mdbVersion 4.2
```

Comment:
Shouldn't the mdbVersion be higher than 4.2 since we are touching this file now?